turbine/src/sigverify_shreds.rs

```diff
@@ -175,18 +218,64 @@ fn run_shred_sigverify<const K: usize>(
     Ok(())
 }
 
+#[must_use]
+fn verify_retransmitter_signature(
```

Comment:
Might be more clear (at leas the return value) if this is called something like `retransmitter_signature_is_valid`

Reply:
We use `verify` in many other places when doing signature verifications, so this might be more consistent to use the same term/style; e.g.

https://github.com/anza-xyz/agave/blob/ba2ff5384/sdk/src/signature.rs#L53
https://github.com/anza-xyz/agave/blob/ba2ff5384/perf/src/sigverify.rs#L113